packages/bpk-mixins/src/mixins/_badges.scss

```diff
@@ -36,13 +36,8 @@ $bpk-spacing-v2: true;
   display: inline-flex;
   padding: bpk-spacing-sm() bpk-spacing-md();
   align-items: center;
-  color: $bpk-text-on-light-day;
+  color: $bpk-text-primary-day;
 
-  @include bpk-themeable-property(
```

Comment:
shouldn't we add a default background-color `background-color: $bpk-surface-highlight-day;` so that by default it would be the colour of the `normal` badge?